backend/src/models/item/postgres.js

```diff
@@ -229,6 +229,55 @@ const ItemRepo = (postgres) => {
     }
   };
 
+
+  // SQL to get list of items by seller_id
+  const getItemsForSellerSQL = `
+    SELECT 
+      items.*, 
+      COALESCE(SUM(bids.total_cost),0) AS current_ledger,
+      CAST(COALESCE(SUM(bids.ticket_count),0) AS integer) AS tickets_sold
+    FROM items LEFT JOIN bids 
+      ON items.item_id = bids.item_id 
+    WHERE items.seller_id = $1
+    GROUP BY items.item_id;`
+
+  // Gets all items that are sold by the user with seller_id
+  const getItemsForSeller = async(seller_id) => {
+    const values = [seller_id]
+    try {
+      const client = await postgres.connect();
+      const res = await client.query(getItemsForSellerSQL, values);
+      client.release();
+      return [res.rows, null];
+    } catch (err) {
+      return [null, err];
+    }
+  };
+
+  // SQL to get list of bids by user_id
+  const getBidsForUserSQL = `
+    SELECT 
+      items.*, 
+      COALESCE(SUM(bids.total_cost),0) AS total_cost,
+      CAST(COALESCE(SUM(bids.ticket_count),0) AS integer) AS tickets_bought
```

Comment:
I feel like we don't want these to be sums? Will this return the total number of tickets bought across all items? Or just the single number from the one item?

I think it can just be 
`SELECT items.*, bids.total_cost, bids.ticket_count AS tickets_bought`

Reply:
They're used as sums because a user might have multiple bids for one item, and I think it would make the most sense to return the total amount of tickets that the user has bought/total price. Since we are joining on the item_id and using the user_id to search, it only gets the number of tickets for the particular item.